src/libwrap/Makefile.am

```diff
@@ -1,18 +1,43 @@
 lib_LTLIBRARIES = libgrate-wrap.la
 
+libgrate_wrap_la_CFLAGS = -pthread \
+	-I$(top_srcdir)/include \
+	-I$(top_srcdir)/src/libgrate
+
 if ENABLE_RNN
-libgrate_wrap_la_CPPFLAGS = -DENABLE_RNN
+libgrate_wrap_la_CFLAGS += -DENABLE_RNN
```

Comment:
What's this about? This is a CPP (C pre-processor) flag, and CPPFLAGS seems like the right place for it...

Reply:
Yes, it is a CPP flag. Do we have any CPP code? Although, I don't know what's the actual difference if CPP flags are used for C as well. Why should we prefer CPPFLAGS over CFLAGS?

Reply:
I wonder if you're confusing CPPFLAGS and CXXFLAGS. CPPFLAGS is for the preprocessor flags. It can (in theory at least) get applied in more places, for instance if we're comiling C++ or non-pre-processed ASM code. It's not a big deal or anything, it just seems strange to me to change this, as CPPFLAGS seems slightly more correct.

Reply:
I thought about C++, indeed.

Reply:
CPPFLAGS returned.